content/stats/view.md

```diff
@@ -102,7 +102,7 @@ public class JavaSnippet {
 
             View.create(Name.create("myapp/lines_in"),
                         "The number of lines that were received",
-                        M_LATENCY_MS,
```

Comment:
Please revert this change, `M_LATENCY_MS` with a count aggregation provides metrics for number of lines read in, assuming `M_LATENCY_MS` records latency once, if not, please use `M_BYTES_IN`.